rdkit/Chem/EnumerateStereoisomers.py

```diff
@@ -298,6 +298,7 @@ def EnumerateStereoisomers(m, options=StereoEnumerationOptions(), verbose=False)
 
     """
   tm = Chem.Mol(m)
+  tm.SetProp('_MolFileChiralFlag', '1')
```

Comment:
I think we only want to do this if there actually is a chiral center, so maybe this needs to move down to around line 311